addons/api/mirage/generated/factories/host-catalog.js

```diff
@@ -5,19 +5,10 @@ import { random, date, datatype } from 'faker';
  * GeneratedHostCatalogModelFactory
  */
 export default Factory.extend({
-  type: () => random.arrayElement(['static', 'plugin']),
+  type: 'static',
```

Comment:
This `type` will be override by `mirage/factories/host-catalog` right? if so, perhaps we should not declared here and let the factory do it? I would like to know your logic on this since I have same casses in other tasks :smiley:

Reply:
We should have some kind of "ground state" default type for typed resources.  The most sensible default for host catalog is `static`.  It's unclear though if this could feasibly be generated.  We'll have to address the problem of _type discovery_ down the road.

Reply:
Ahh ok that makes sense. I will apply this approach on the PR's I have, thanks for the insights.